.gitignore

```diff
@@ -1,7 +1,31 @@
-# Don't commit
+# Logs
+logs
+*.log
 
-# Installed Node Modules. Commit the package.json instead
-node_modules/
+# Runtime data
+pids
+*.pid
+*.seed
 
-# Bot Config: Contains highly sensitive data, NEVER Commit
-newxkitbot.conf.js
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
```

Comment:
Looks like this is a (partial?) copy-paste of [GitHub's Node.gitignore](https://github.com/github/gitignore/blob/da00310ccba9de9a988cc973ef5238ad2c1460e9/Node.gitignore)